src/compiler/objective_c_plugin.cc

```diff
@@ -72,7 +77,20 @@ class ObjectiveCGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
       for (int i = 0; i < file->dependency_count(); i++) {
         ::grpc::string header = grpc_objective_c_generator::MessageHeaderName(
             file->dependency(i));
-        proto_imports += ::grpc::string("#import \"") + header + "\"\n";
+        const grpc::protobuf::FileDescriptor *dependency = file->dependency(i);
+        if (IsProtobufLibraryBundledProtoFile(dependency)) {
+          ::grpc::string base_name = header;
+          grpc_generator::StripPrefix(&base_name, "google/protobuf/");
+          proto_imports +=
+            "#if GPB_USE_PROTOBUF_FRAMEWORK_IMPORTS\n"
+            "  #import <" + ::grpc::string(ProtobufLibraryFrameworkName) +
+            "/" + base_name + ">\n"
+            "#else\n"
+            "  #import \"" + header + "\"\n"
+            "#endif\n";
+       } else {
```

Comment:
Ok I now saw what it is: the `} else {` and the `}` that closes the block need an extra space in front.